<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1584</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List visualization</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	background-color: 	#03958A;
	color: #ffffff;
	text-transform:uppercase;
	border: none;
	outline:none;
	font-size: 30px;
	width: 100px;
	height: 40px;
}

QPushButton:hover{
	background-color: #02665e;
}

QPushButton:pressed{
	background-color: #00423c;
}

QSpinBox{
	font-size: 30px;
}

QLabel{
	font-family:  sans-serif;;
	font-size: 30px;
}</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>1900</width>
      <height>431</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBackButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Push back</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pushBackField">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="insertValueField">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="valueLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="positionLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>110</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="insertPositionField">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>20</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>20</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: 	#03958A;
	color: #ffffff;
	text-transform:uppercase;
	border: none;
	outline:none;
	font-size: 30px;
	width: 100px;
	height: 40px;
}

QPushButton:hover{
	background-color: #02665e;
}

QPushButton:pressed{
	background-color: #00423c;
}</string>
    </property>
    <property name="text">
     <string>Erase</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="erasePositionField">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>20</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="positionLabel_2">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>20</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: 	#03958A;
	color: #ffffff;
	text-transform:uppercase;
	border: none;
	outline:none;
	font-size: 30px;
	width: 100px;
	height: 40px;
}

QPushButton:hover{
	background-color: #02665e;
}

QPushButton:pressed{
	background-color: #00423c;
}</string>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redoButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>240</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:red;
}
QPushButton:hover{
background-color:#8b0000;
}</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1584</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
